examples/patterns/configuring_experiments/conf/config_with_override.yaml

```diff
@@ -0,0 +1,4 @@
+defaults:
+  - db: mysql
+  - server: apache
+  - override hydra/env: custom
```

Comment:
The default list tests are sufficient.
We should avoid complicating examples for the purpose of testing.
Can you undo the patterns changes?

Reply:
sounds good, removed.